rfcs/password_validation_limits.md

```diff
@@ -56,15 +63,20 @@ When executed:
 - calls `zxcvbn` to obtain a complexity of passwords and returns whether the given password matches policy
 
 Also, we can pass user-provided data into `zxcvbn` to check whether some sensitive data used in the password.
+
 ### Sample config
-`forceCheckFieldName` and `inputFieldNames` values point to parent object field
+`skipCheckFieldName` - Allows skipping password check if any field exists.
+`forceCheckFieldName` - Allows forcing password check if any field exists.
+`inputFieldNames` - values point to parent object fields that passed into `zxcvbn`.
+
 `enabled` - disable/enable validator. Default value is false.
 ```js
 const config = {
   enabled: false,
   minStrength: 3, // Desired strength
-  forceCheckFieldName: ['checkPassword'], // force enable password to check if the object field value set..
-  inputFieldNames: [ // Linked fields list, allow filter case sensitive data in the password from the parent object.
+  skipCheckFieldNames: ['skipPassword'], // Force disable password to check if the object field value exists.
+  forceCheckFieldNames: ['checkPassword'], // Force enable password check if the object field value exists.
```

Comment:
How about "forces password check <...>" and "disables password check <...>" (one line above)?